build_html.sh

```diff
@@ -24,7 +24,7 @@ do
         printf '/:version/*    /:version/en/:splat    301\n' >> build/html/_redirects
     else
         printf '/:version/%s/* /:version/%s/404.html  404\n' "$lang" "$lang" >> build/html/_redirects
-        printf '/:version/*    /:version/%s/:splat    301 Language=%s\n' "$lang" "$lang" >> build/html/_redirects
+        printf '/:version/*    /:version/%s/:splat    301 Language=%s\n' "$lang" "${lang/_/-}" >> build/html/_redirects
```

Comment:
isn't this the wrong way around. netflify languages contain a dash but our pages contain underscores?

Reply:
In the sphinx config we use now the underscores. This one replaces the underscores with dashes for Netlify so I think it should work.